NEWS.md

```diff
@@ -84,6 +84,12 @@
 
 * Increase the default `nbin` of `guide_colourbar()` to place the ticks more precisely (#3508, @yutannihilation).
 
+* Modified `manual_scale()` to match `values` with the order of `breaks`
```

Comment:
Sorry, one last nitpick: I think it's better to write "`manual_scale()` now matches ..." to emphasize the new behavior rather than the fact the code was modified.